### DISCUSSION TITLE
Welcome to OsmAnd Discussions!

### DISCUSSION THREAD

**Discussion by vshcherb** [osmandapp/OsmAnd#10910] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jfburke3** [comment]
Hi _waves to everyone_. I'm Jack, and I like to find new and exciting ways to break software. And sometimes I just use it for its intended purpose.

**Reply by vshcherb** [reply]
That's quite intense.

**Reply by jfburke3** [reply]
It keeps me off the street and out of jail. 😁

**Comment by barteis** [comment]
Hi!  I'm here to ask questions, answer some, and learn from others.
- Here's a list of my GPS video tutorials, mostly on OsmAnd: https://sites.google.com/view/bartetutorials/home
- Here's a profile: https://osmand.net/blog/story-19
